backend/src/main/scala/cromwell/backend/backend.scala

```diff
@@ -84,6 +84,18 @@ case class BackendWorkflowDescriptor(id: WorkflowId,
 
   val rootWorkflow = breadCrumbs.headOption.map(_.callable).getOrElse(callable)
   val rootWorkflowId = breadCrumbs.headOption.map(_.id).getOrElse(id)
+  lazy val hogGroup = {
+    val config = ConfigFactory.load
```

Comment:
please take `hogGroupField` as `Option[String]` instead of statically loading Config

Reply:
I'm interpreting this as "it would be nice to make this better, but now isn't necessarily the time". Let me know if that's wasn't your intention.